corehq/apps/change_feed/data_sources.py

```diff
@@ -16,6 +17,14 @@ def get_document_store(data_source_type, data_source_name):
             if settings.DEBUG:
                 return None
             raise
+    elif data_source_type == POSTGRES:
+        conn = psycopg2.connect(
+            database='commcarehq',
+            user='commcarehq',
+            password='commcarehq',
+        )
+        conn.set_isolation_level(psycopg2.extensions.ISOLATION_LEVEL_AUTOCOMMIT)
+        return conn
```

Comment:
This should return an instance of [DocumentStore](https://github.com/dimagi/commcare-hq/blob/5178ce0e0a6ca30de2efc7dd4fd0f836f3dc5c90/corehq/ex-submodules/pillowtop/dao/interface.py#L4-L4)